ingestion/functions/parsing/mexico/input_event.json

```diff
@@ -0,0 +1,7 @@
+{
+    "env": "local",
+    "s3Bucket": "epid-sources-raw",
+    "s3Key": "5f59e8c47ef66004bc9c91ce/2020/09/10/0855/content.csv",
+    "sourceUrl": "http://datosabiertos.salud.gob.mx/gobmx/salud/datos_abiertos/datos_abiertos_covid19.zip",
```

Comment:
is the original source really a zip?? in which case we'll have to implement unzipping before parsing. (worth filing an issue to track that if that's true)

Reply:
Yes unfortunately it is! Chatted it over with AB a little, yeah we'll either have to implement that or rehost every time it is updated. I'll file an issue.